<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulação de Financiamento</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(23, 23, 29);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(34, 34, 44);
	border: 2px solid rgb(37,39,48);
	border-radius: 20px;
	color: #fff;
	padding-left: 20px
}	
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Renda Mensal: R$</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(34, 34, 44);
	border: 2px solid rgb(37,39,48);
	border-radius: 20px;
	color: #fff;
	padding-left: 20px
}	
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Valor do imovel: R$</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(34, 34, 44);
	border: 2px solid rgb(37,39,48);
	border-radius: 20px;
	color: #fff;
	padding-left: 20px
}	
QLineEdit:hover{
	border: 2px solid rgb(48, 50, 62);
}
QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	background-color: rgb(43, 45, 56);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="placeholderText">
     <string>Anos: ex. 25</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Preencha os campos para fazer sua simulação</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(58, 143, 33);
	border: 2px solid rgb(37,39,48);
	border-radius: 20px;
	color: #fff;
	padding-left: 1px
}	
QPushButton:hover{
	border: 2px solid rgb(136, 138, 133);
}
QPushButton:focus{
	border: 2px solid rgb(37, 39, 48);
	
	background-color: rgb(69, 168,  83);
}




</string>
    </property>
    <property name="text">
     <string>ENVIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(32, 74, 135);
	border: 2px solid rgb(37,39,48);
	border-radius: 20px;
	color: #fff;
	padding-left: 1px
}	
QPushButton:hover{
	border: 2px solid rgb(136, 138, 133);
}
QPushButton:focus{
	border: 2px solid rgb(37, 39, 48);
	background-color: rgb(114, 159, 207);
}




</string>
    </property>
    <property name="text">
     <string>IMPRIMIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-10</y>
      <width>131</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagens/bank.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
